chaoscenter/web/src/components/StatusHeatMap/StatusHeatMap.tsx

```diff
@@ -129,7 +129,7 @@ export function StatusHeatMap(props: StatusHeatMapProps): React.ReactElement {
                   {execution.executedBy?.username !== 'pipeline' ? (
                     <>
                       {execution.executedBy?.username ?? getString('chaosController')}
-                      {execution.executedBy?.username && ' | Manually'}
+                      {execution.executedBy?.username && getString('manually')}
```

Comment:
```suggestion
                      {execution.executedBy?.username && `| ${getString('manually')}`}
```
NIT: Update strings file to just contain the text and then use in the component.